src/Constraint/DraftFour/Type.php

```diff
@@ -13,6 +13,31 @@
     const KEYWORD = 'type';
 
     /**
+     * Whether examples like 98249283749234923498293171823948729348710298301928331
+     * and "98249283749234923498293171823948729348710298301928331" are valid strings.
+     */
+    const BIGINT_MODE_STRING_VALID = 1;
+    const BIGINT_MODE_STRING_INVALID = 2;
+
+    /**
+     * @var int
+     */
+    private static $defaultBigintMode = self::BIGINT_MODE_STRING_INVALID;
```

Comment:
The ruleset binds the instance as a singleton, so you should only have to set it once per ruleset instance.

```php
$ruleset = new DraftFour();

$ruleset->get(Type::KEYWORD)->setBigIntMode(TYPE::BIGINT_MODE_STRING_VALID);

$validator = new Validator($data, $schema, $ruleset);
$validator = new Validator($otherData, $otherSchema, $ruleset);
```

It's not as convenient as the static parameter but it's consistent with the the other constraints that accept configuration parameters.